src/Components/CourseCart.js

```diff
@@ -6,16 +6,23 @@ const generateCourseList = (selectedCourses, removeCourseFromCart) => {
     return selectedCourses.map((course, index) => {
       return (
           <React.Fragment>
-            <span
-                className={"fake-link"}
+            {/*change onClick to display information about the course*/}

```

Comment:
Ah, all this stuff that was once in the span should be moved into its own component (ie a const arrow function that returns one React.Fragment).  It will help a lot with readability!